doc/API.rst.txt

```diff
@@ -158,29 +158,27 @@ change is made.
 .. toctree::
    :maxdepth: 1
    :hidden:
-
-   main_namespace
-   cluster
-   constants
-   datasets
-   fft
-   fftpack
-   integrate
-   interpolate
-   io
-   linalg
-   misc
-   ndimage
-   odr
-   optimize
-   signal
-   sparse
-   sparse.linalg
-   sparse.csgraph
-   spatial
-   special
-   stats
-   ccallback
+   :titlesonly:
+
+   scipy <main_namespace>
+   scipy.cluster <cluster>
+   scipy.constants <constants>
+   scipy.datasets <datasets>
+   scipy.fft <fft>
+   scipy.fftpack <fftpack>
+   scipy.integrate <integrate>
+   scipy.interpolate <interpolate>
+   scipy.io <io>
+   scipy.linalg <linalg>
+   scipy.misc <misc>
+   scipy.ndimage <ndimage>
+   scipy.odr <odr>
+   scipy.optimize <optimize>
+   scipy.signal <signal>
+   scipy.sparse <sparse>
+   scipy.spatial <spatial>
+   scipy.special <special>
+   scipy.stats <stats>
```

Comment:
Looks like everything is here except what was intended to be removed (sub-subpackages, ccallback).